### ISSUE TITLE
Cross attention

### ISSUE THREAD

**Issue by jerryxfu** [jerryxfu/medive#6]
[MultiheadAttention](https://docs.pytorch.org/docs/stable/generated/torch.nn.MultiheadAttention.html)

- forward pass
- adds to 1 (softmax)
- attention masking (for padding)
